lib/Jsrt/JsrtCommonExports.inc

```diff
@@ -115,5 +115,4 @@
     JsRunSerialized
     JsCreatePropertyId
     JsCopyPropertyId
```

Comment:
Unrelated... was searching where "JsDiagEvaluate" was exported, found in "bin/ChakraCore/ChakraCore.def". Looks these are supposed to be in that file as well?